acceleration/distributed_training/distributed_training.md

```diff
@@ -0,0 +1,31 @@
+## Data Preparation
+
+Users need to download the dataset Task01_BrainTumour from the MICCAI challenge [Medical Segmentation Decathlon](http://medicaldecathlon.com/) for the following examples.
+
+## Multi-GPU Training
+
+Users can set your `NUM_GPUS_PER_NODE`, `NUM_NODES`, `INDEX_CURRENT_NODE`, as well as `DIR_OF_TESTDATA` for the directory of the test dataset.
```

Comment:
Does the test dataset mean the entire dataset (Task01_BrainTumour) to the imagesTs?

Reply:
I think the `DIR_OF_TESTDATA` here just means that the data used for this tutorial. Not real "testing data".

Reply:
Updated to `DIR_OF_DATA`.